src/index.js

```diff
@@ -1,6 +1,16 @@
-const fibonacci = (n) => {
+// Generators are a special class of functions that simplify the task of writing iterators.
+function* fib(n, current = 1, next = 1) {
+ // if (n === 0) {
+   // return 0;
+  // }
+  if (n === 0) {
+    return 1;
```

Comment:
when you use generators as iterables, return value isn't nessesary. You can use `return;` or 
```js
if ( n >0) {
  yield current;
  yield* fib(n - 1, next, current + next);
}
```
When js can't find another yield, the `next` function return `{value: undefined, done: true}`. 
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols